tests/test_builders/xpath_util.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import re
+from typing import TYPE_CHECKING
+from xml.etree.ElementTree import tostring
+
+if TYPE_CHECKING:
+    import os
+    from collections.abc import Callable, Sequence
+    from xml.etree.ElementTree import Element
+
+
+def _gettext(node: Element) -> str:
```

Comment:
```suggestion
def _get_text(node: Element) -> str:
```

`get_text` != [`gettext`](https://www.gnu.org/software/gettext/)

Reply:
Ah yes, well technically it's not really an issue since it's a private method but yeah it could be confusing.